src/asset/asset-list-loader.js

```diff
@@ -0,0 +1,214 @@
+import { EventHandler } from '../core/event-handler.js';
+
+import { Asset } from './asset.js';
+
+/**
+ * @class
+ * @name AssetListLoader
+ * @augments EventHandler
+ * @classdesc Used to load a group of assets and fires a callback when all assets are loaded.
+ * @param {Asset[]|number[]} assetList - An array of {@link Asset} objects to load or an array of Asset IDs to load.
+ * @param {AssetRegistry} assetRegistry - The application's asset registry.
+ */
```

Comment:
This is the 'old-style' JSDoc comment block. Look at other classes to see how we do this now. 😄